src/cloudlabs/forms/customise_setup.py

```diff
@@ -6,6 +6,6 @@
 class CustomiseSetupForm(FlaskForm):
     id = HiddenField()
     setup_script = TextAreaField('Setup script',
-                                 description='Specify the (bash) script run \
-                                              to initialise the new host',
+                                 description='Specify the (bash) script run'
+                                             'to initialise the new host',
```

Comment:
You need to add a space at the start of this second string, otherwise the words get run together.